src/components/molecules/ImageInput/ImageInput.tsx

```diff
@@ -24,94 +24,82 @@ interface ImageInputProps extends ContainerClassName {
   register: ReturnType<typeof useForm>["register"];
 }
 
-export const ImageInput = React.forwardRef<HTMLInputElement, ImageInputProps>(
-  (
-    {
-      image,
-      remoteUrlInputName,
+export const ImageInput: React.FC<ImageInputProps> = ({
+  image,
+  remoteUrlInputName,
+  remoteImageUrl,
+  containerClassName,
+  imageClassName,
+  name,
+  error,
+  disabled,
+  setValue,
+  register,
+}) => {
+  const imageUrl = useMemo(
+    () =>
+      (image && image.length > 0 && URL.createObjectURL(image[0])) ||
```

Comment:
```suggestion
      (image.?length > 0 && URL.createObjectURL(image[0])) ||
```

Reply:
No, you can't do that as the type of `image?.length` now includes `undefined` and cannot be used inside a boolean expression.